### PR TITLE
Remove Unused files and helpers referencing SVB

### PR DESCRIPTION
**Pull request by aramshiva**
Fixes #4984.
Removes unused SVB files/helpers

### CODE REVIEW

**Comment by aramshiva** [comment]
I'll do some extra checks once preview is deployed as don't want to make sure I broke something.

**Comment by aramshiva** [comment]
ScreenShot 2024-03-27 at 22 25 09@2x
@garyhtou can you provide logs?

**Comment by garyhtou** [comment]
@cjdenio when you get a chance, could you review this?

File: app/services/transaction_engine/transaction_csv_service/upload.rb [hunk 1]
```diff
@@ -10,10 +10,6 @@ def initialize(transaction_csv_id:)
       def run
         transaction_csv.mark_processing!
 
-        transaction_csv.file.open do |f|
```
Comment (cjdenio) [inline app/services/transaction_engine/transaction_csv_service/upload.rb]: This service should be completely removed— deleting these three lines just makes it do nothing

**Review by cjdenio — CHANGES_REQUESTED** [review]
I see the test for `Parsers::SiliconValleyBank` was removed, but the service itself was not— we should be removing both at the same time

**Review by cjdenio — APPROVED** [review]

**Pull request merged**